.github/workflows/ci.yml

```diff
@@ -394,7 +395,7 @@ jobs:
           # | pkg2 | test2 | 4 | race |
           # | pkg3 | test3 | 6 | fips |
 
-          jq -r -n 'inputs | select(.Action == "fail") | select(.Test != null) | "| \(.Package) | \(.Test) | \(input_filename | split("-") | .[2] | sub(".json"; "")) | \(input_filename | split("-") | .[3] // "normal" | sub(".json";"") )"' failure-summary-*.json | sort >> "$temp_file_name"
+          jq -r -n 'inputs | select(.Action == "fail") | "| \(.Package) | \(.Test // "-") | \(input_filename | split("-") | .[2] | sub(".json"; "")) | \(input_filename | split("-") | .[3] // "normal" | sub(".json";"") )"' failure-summary-*.json | sort >> "$temp_file_name"
```

Comment:
Changed that to avoid false positives. Sometimes we get a failure with no test associated with it, so need to keep these entries in.